### PR TITLE
[Microsoft Teams] add microsoft-teams-graph-auth-reset command

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8718)

## Description
Add microsoft-teams-graph-auth-reset command.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 1]
```diff
@@ -142,8 +142,8 @@ configuration:
   section: Connect
   required: false
 description: Send messages and notifications to your team members.
-display: Microsoft Teams
-name: Microsoft Teams
+display: Microsoft Teams test
+name: Microsoft Teams test
```
Comment (michal-dagan) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml:146]:
```suggestion
name: Microsoft Teams
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 2]
```diff
@@ -142,8 +142,8 @@ configuration:
   section: Connect
   required: false
 description: Send messages and notifications to your team members.
-display: Microsoft Teams
-name: Microsoft Teams
+display: Microsoft Teams test
```
Comment (michal-dagan) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml:145]:
```suggestion
display: Microsoft Teams
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 3]
```diff
@@ -265,16 +265,20 @@ Note: The [microsoft-teams-ring-user](https://learn.microsoft.com/en-us/graph/ap
 2. Enter your Client/Application ID in the *Bot ID* parameter. 
 3. Enter your Client Secret in the *Bot Password* parameter.
 4. Save the instance.
+5. [Add the Demisto Bot to a Team](#Add-the-Demisto-Bot-to-a-Team)
```
Comment (merit-maita) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:268]: for client credentials flow, no need to add the bot to the team to test the instance, totally fine (and makes sense) and test it right after configuring the instance

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 4]
```diff
@@ -265,16 +265,20 @@ Note: The [microsoft-teams-ring-user](https://learn.microsoft.com/en-us/graph/ap
...
 2. Enter your Client/Application ID in the *Bot ID* parameter. 
 3. Enter your Client Secret in the *Bot Password* parameter.
 4. Enter your Application redirect URI in the *Application redirect URI* parameter.
-5. Run the ***!microsoft-teams-generate-login-url*** command in the War Room and follow the instructions.
-6. Save the instance.
-7. Run the ***!microsoft-teams-auth-test*** command. A 'Success' message should be printed to the War Room.
+5. Save the instance.
+6. [Add the Demisto Bot to a Team](#Add-the-Demisto-Bot-to-a-Team)
+7. Run the ***!microsoft-teams-generate-login-url*** command in the War Room and follow the instructions.
+8. Save the instance.
```
Comment (merit-maita) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:280]: it's mentioned twice.

**Review by merit-maita — COMMENTED** [review]
it looks great!, just 2 comments.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 4 above)
Comment (michal-dagan) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:280]: it's on purpose

**Review by merit-maita — APPROVED** [review]
good job!

**Pull request merged**